Dockerfile

```diff
@@ -16,6 +16,9 @@ LABEL name="openshift-ansible" \
 
 USER root
 
+# Create a symlink to /opt/app-root/src so that files under /usr/share/ansible are accessible
+RUN mkdir -p /usr/share/ansible/ && ln -s /opt/app-root/src /usr/share/ansible/openshift-ansible
+
```

Comment:
Why is this needed?

The "productized" container image built by `Dockerfile.rhel7` has the roles/playbooks under `/usr/share/ansible/openshift-ansible` because they are installed from the RPM, but I believe this is not needed in the upstream image. You can/should  use relative paths e.g. for `PLAYBOOK_FILE` and these would work the same in both cases, no?